functions/PushNotificationTutorConfirmation.js

```diff
@@ -0,0 +1,111 @@
+const {onDocumentUpdated} = require("firebase-functions/v2/firestore");
+const {setGlobalOptions} = require("firebase-functions/v2");
+const {initializeApp} = require("firebase-admin/app");
+const {getFirestore} = require("firebase-admin/firestore");
+const admin = require("firebase-admin");
+
+initializeApp();
+
+setGlobalOptions({
+  maxInstances: 10,
+  timeoutSeconds: 60,
+  memory: "256MiB"});
+
+exports.pushNotificationTutorLessonConfirmation = onDocumentUpdated(
+    "lessons/{lessonId}",
+    async (event) => {
+      const before = event.data.before.data(); // Data before the update
+      const after = event.data.after.data(); // Data after the update
+      const lessonId = event.params.lessonId;
+      const db = getFirestore();
```

Comment:
Great adjustment of the naming as it is clearer now than at the beggining of the week, great improvement